eyeshade/controllers/reports.js

```diff
@@ -63,7 +63,8 @@ v3.publishers = {}
 v1.publisher.contributions = {
   handler: (runtime) => {
     return async (request, reply) => {
-      const authority = request.auth.credentials.provider + ':' + request.auth.credentials.profile.username
+      // const authority = request.auth.credentials.provider + ':' + request.auth.credentials.profile.username
```

Comment:
i think that `git` is able to keep track of what gets added/deleted/changed, so i'd much rather not see lines commented out throughout this commit...